PendingReleaseNotes

```diff
@@ -39,6 +39,11 @@
   The default ssl options setting is "no_sslv2:no_sslv3:no_tlsv1:no_tlsv1_1".
   If you want to return back the old behavior add 'ssl_options=' (empty) to
   ``rgw frontends`` configuration.
+* RGW: The behavior for Multipart Upload was: `POST` notification is sent in the 
+  beginning of the multipart upload and `PUT` notification sent for each of the 
```

Comment:
please change to:
RGW: The behavior for Multipart Upload was modified so that only `CompleteMultipartUpload` notification is sent at the end of the multipart upload. The `POST` notification at the beginning of the upload, and `PUT` notifications that were sent on each part are not sent anymore.